<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabGeneral">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="toolTip">
           <string>Atlantian subgroup information for which this report is for.</string>
          </property>
          <property name="toolTipDuration">
           <number>3</number>
          </property>
          <property name="title">
           <string>Group Information</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="toolTip">
              <string>The Atlantian subgroup is this report for.</string>
             </property>
             <property name="text">
              <string>Group Name*</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="q4_whichGroup"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="toolTip">
              <string>Quarter for which this report spans.</string>
             </property>
             <property name="text">
              <string>Quarter*</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="q5_whichQuarter"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="toolTip">
              <string>Year this report is for.</string>
             </property>
             <property name="text">
              <string>Year*</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="q6_whatYear">
             <property name="whatsThis">
              <string>Year of report</string>
             </property>
             <property name="inputMask">
              <string>D999</string>
             </property>
             <property name="maxLength">
              <number>4</number>
             </property>
             <property name="placeholderText">
              <string>1970</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="toolTip">
           <string>General contact information for the person submitting this report.</string>
          </property>
          <property name="toolTipDuration">
           <number>3</number>
          </property>
          <property name="title">
           <string>Contact Information</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="toolTip">
              <string>The reporter's position in the group.</string>
             </property>
             <property name="text">
              <string>Position*</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="q41_whatIs"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="toolTip">
              <string>Reporter's chosen SCA name.</string>
             </property>
             <property name="text">
              <string>SCA Name*</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="toolTip">
              <string>Reporter's mundane name.</string>
             </property>
             <property name="text">
              <string>Modern Name*</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="toolTip">
              <string>Reporter's email address.</string>
             </property>
             <property name="text">
              <string>E-mail Address*</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="q9_scaName">
             <property name="placeholderText">
              <string>SCA Name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="q10_modernName">
             <property name="placeholderText">
              <string>Mundane Name</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="q18_emailAddress">
             <property name="placeholderText">
              <string>someone@example.com</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_10">
             <property name="toolTip">
              <string>Whether the reporter has a copy of their warrant for their position within the group.</string>
             </property>
             <property name="text">
              <string>Do you have a copy of your warrant?*</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QRadioButton" name="warrantYes">
               <property name="text">
                <string>Yes</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_3</string>
               </attribute>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="warrantNo">
               <property name="text">
                <string>No</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_3</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="q19_membershipNumber">
             <property name="inputMask">
              <string>9000000</string>
             </property>
             <property name="maxLength">
              <number>7</number>
             </property>
             <property name="placeholderText">
              <string>123456</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_8">
             <property name="toolTip">
              <string>Reporter's SCA Membership number.</string>
             </property>
             <property name="text">
              <string>Membership Number*</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_22">
             <property name="toolTip">
              <string>Reporter's SCA Membership expiration date.</string>
             </property>
             <property name="text">
              <string>Membership Expiration*</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QDateEdit" name="memberExpireDateEdit">
             <property name="displayFormat">
              <string>d MMM yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="date">
              <date>
               <year>2021</year>
               <month>3</month>
               <day>1</day>
              </date>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="3">
           <widget class="QCheckBox" name="phoneShare">
            <property name="text">
             <string>Phone Number</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="toolTip">
             <string>Which of these, if any, do you give persmission to share publicly?</string>
            </property>
            <property name="text">
             <string>Share Permissions?</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="mailingShare">
            <property name="text">
             <string>Mailing Address</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="modernShare">
            <property name="text">
             <string>Modern Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QCheckBox" name="emailShare">
            <property name="text">
             <string>E-mail Address</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabOther">
       <attribute name="title">
        <string>Other</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Seneschal Information</string>
          </property>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Seneschal's SCA Name*</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Seneschal's Email*</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="q42_seneschalsSca"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="q43_seneschalsEmail"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Contact and Deputy Information</string>
          </property>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Do you need to update your contact information?*</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QFormLayout" name="formLayout_6">
             <item row="0" column="0">
              <widget class="QRadioButton" name="contactInfoRadioYes">
               <property name="text">
                <string>Yes</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="contactInfoRadioNo">
               <property name="text">
                <string>No</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Do you have a deputy?*</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QFormLayout" name="formLayout_7">
             <item row="0" column="0">
              <widget class="QRadioButton" name="deputyRadioYes">
               <property name="text">
                <string>Yes</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="deputyRadioNo">
               <property name="text">
                <string>No</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup_2</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Going Forward</string>
          </property>
          <layout class="QFormLayout" name="formLayout_9">
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Is there anyone in your local group that needs recognition?</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Is there anything you need from the KMoAS?</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>What are you group's goals for A&amp;S for the next period?</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPlainTextEdit" name="q36_whatAre"/>
           </item>
           <item row="1" column="1">
            <widget class="QPlainTextEdit" name="q37_isThere37"/>
           </item>
           <item row="0" column="1">
            <widget class="QPlainTextEdit" name="q38_isThere38"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabActivities">
       <attribute name="title">
        <string>Activities</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Workshops and Classes</string>
          </property>
          <layout class="QFormLayout" name="formLayout_12">
           <item row="0" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Workshops and Classses:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPlainTextEdit" name="q29_workshopsamp"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Events</string>
          </property>
          <layout class="QFormLayout" name="formLayout_11">
           <item row="0" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Event A&amp;S:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Collegia or Universities</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPlainTextEdit" name="q32_eventAamps"/>
           </item>
           <item row="1" column="1">
            <widget class="QPlainTextEdit" name="q46_collegiaOr46"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>Miscallaneous A&amp;&amp;S</string>
          </property>
          <layout class="QFormLayout" name="formLayout_10">
           <item row="0" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Misc. A&amp;S Activities:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPlainTextEdit" name="q34_miscAamps"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="submitButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Submit Report</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Settings"/>
    <addaction name="actionExport_Settings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionImport_Settings">
   <property name="text">
    <string>Import Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_Settings">
   <property name="text">
    <string>Export Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
